### PR TITLE
0.2/121 accordion

### PR DESCRIPTION
**Pull request by gingerbreadpup**
Progress on the payment page. Markup needs to be tightened up but will do that tomorrow. Bed time.

### CODE REVIEW

File: src/pages/Payment.vue [hunk 1]
```diff
@@ -2,8 +2,87 @@
...
+                        </div>
+                        <div class="row-code">
+                          <div class="code">RVP12345563</div><div class="date">12/1/17</div>
+                        </div>
+                      </label>
+                    </div>
+                  </div>
+                </div>
+
+              </div>
```
Comment (katjam) [inline src/pages/Payment.vue]: I am guessing these tab rows will be a loop when you have data? @charlesstrange2 Have you defined a json structure you want that data in yet?

File: src/pages/Tap.vue [hunk 2]
```diff
@@ -7,7 +7,6 @@
                 <h1>Type a voucher code</h1>
 
                 <form id="textVoucher" v-on:submit.prevent>
-                    <label for="voucherBox" id="lblVoucherBox">Type a voucher code</label>
```
Comment (katjam) [inline src/pages/Tap.vue:10]: Is it bad form to not have a label at all - even an invisible one for accessibility?

**Review by katjam — CHANGES_REQUESTED** [review]
Looks pretty good to me. Just the label point.

File: src/pages/Payment.vue (hunk 1 above)
Comment (charlesstrange2) [reply src/pages/Payment.vue]: Yup, they'll all be vue'd with for-in statements. JSON is cooking. then SQL/Eloquent to make it happen.

Comment (katjam) [reply src/pages/Payment.vue]: Have you not got the mocks ready for it yet?

**Comment by charlesstrange2** [comment]
nope - been trying to get auth/state working.

File: src/pages/Tap.vue (hunk 2 above)
Comment (gingerbreadpup) [reply src/pages/Tap.vue:10]: It is bad form, yes - meant to put an invisible one back in but I guess I got distracted!

**Review by katjam — APPROVED** [review]
:shipit: :sunflower: Good one!

**Pull request merged**